install_template/deploy.mjs

```diff
@@ -81,8 +81,8 @@ const moveDoc = async (product, platform, version) => {
     sles_12_x86_64: "06",
     "ubuntu_20.04_x86_64": "07",
     "ubuntu_18.04_x86_64": "07a",
-    debian_10_x86_64: "07b",
-    debian_9_x86_64: "08",
+    debian_10_x86_64: "08",
```

Comment:
I thought we decided to fix these issues using the nav option in the front matter?